views/membership.erb

```diff
@@ -0,0 +1,261 @@
+                  <div class="text-center"><span>Stredná firma<i class="glyphicon glyphicon-chevron-down"></i></span></div>
+                </a>
+              </li>
+              <li>
+                <a href="#tab4default" data-toggle="tab">
+                  <div class="text-center"><span>Liga výnimočných<i class="glyphicon glyphicon-chevron-down"></i></span></div>
+                </a>
+              </li>
+              <li>
+                <a href="#tab5default" data-toggle="tab">
+                  <div class="text-center"><span>TOP člen<i class="glyphicon glyphicon-chevron-down"></i></span></div>
+                </a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="panel-body">
+      <div class="container">
+        <div class="row">
+          <div class="col-md-14">
+            <div class="tab-content">
+            <div class="tab-pane fade" id="tab1default">
+                <div class="row">
+                  <div class="col-md-3"></div>
+                  <div class="col-md-9 tab-pane-content">
+                    <div class="row">
+                      <div class="col-md-6">
+                        <h3>Benefity</h3>
+                        <p>Kategória určená pre fyzické osoby:</p>
+                        <ul>
+                          <li><p>Účasť na chode o.z.</p></li>
+                          <li><p>Meno v zozname členov</p></li>
+                          <li><p>Networking akcie</p></li>
+                          <li><p>Informovanosť o eGovernmente</p></li>
+                        </ul>
+                      </div>
+                      <div class="col-md-3">
+                        <h3>Ekosystém</h3>
+                        Datahub <p>/</p>
+                        Autoform <p>/</p>
+                        GovBox: <p>/</p>
+                      </div>
+                      <div class="col-md-3">
+                        <h3>Ročné členstvo</h3>
+                        <strong>100 euro</strong>
+                      </div>
+                      <div class="col-md-12">
+                      </div>
+                    </div>
+                  </div>
+                </div>
+              </div>
+              <div class="tab-pane fade in active" id="tab2default">
+                <div class="row">
+                  <div class="col-md-3"></div>
+                  <div class="col-md-9 tab-pane-content">
+                    <div class="row">
+                      <div class="col-md-6">
+                        <h3>Benefity</h3>
+                        <p>Ako základný člen, no naviac:</p>
+                        <ul>
+                            <li><p>Logo na web stránke</p></li>
+                        </ul>
+                      </div>
+                      <div class="col-md-3">
+                        <h3>Ekosystém</h3>
+                        Datahub <p>1024 requestov</p>
```

Comment:
1024 za ake obdobie?

Potom [inde](https://ekosystem.slovensko.digital/otvorene-api#rate-limit) volame requesty 'pristupy'. Tak to rasdej dajme po slovensky aj tu.